<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5" rowstretch="80,24" rowminimumheight="0,0">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="50,40,55">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="10,6">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Find Patient </string>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>6</x>
              <y>40</y>
              <width>251</width>
              <height>24</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="labelPatientId">
               <property name="text">
                <string>Patient ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPatientIdFound"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonFindPatient">
               <property name="text">
                <string>Find</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QTextEdit" name="textEditPatientData">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>70</y>
              <width>251</width>
              <height>181</height>
             </rect>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Registration Patient</string>
           </property>
           <widget class="QPushButton" name="pushButtonRegisterNewPatient">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>130</y>
              <width>80</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Creat</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditPatientNameReg">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>30</y>
              <width>71</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditPatieantIDReg">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>30</y>
              <width>71</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>48</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Patient ID</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>148</x>
              <y>30</y>
              <width>27</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>35</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Gender</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>51</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Birth Day</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButtonMaleGender">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>60</y>
              <width>51</width>
              <height>19</height>
             </rect>
            </property>
            <property name="text">
             <string>Male</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButtonFemaleGender">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>60</y>
              <width>61</width>
              <height>19</height>
             </rect>
            </property>
            <property name="text">
             <string>Female</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButtonOtherGender">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>60</y>
              <width>84</width>
              <height>19</height>
             </rect>
            </property>
            <property name="text">
             <string>Other</string>
            </property>
           </widget>
           <widget class="QDateTimeEdit" name="dateTimeBirthDay">
            <property name="geometry">
             <rect>
              <x>73</x>
              <y>90</y>
              <width>181</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Patient Needs</string>
         </property>
         <widget class="QTextEdit" name="textEditDoctorDef">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>201</width>
            <height>181</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Doctor Definition</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>410</y>
            <width>80</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Patient ID:</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelPatientID">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>40</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>270</y>
            <width>102</width>
            <height>131</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxSurgeryNeed">
             <property name="text">
              <string>Surgery Need</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxDentisNeed">
             <property name="text">
              <string>Dentis Need</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxRadiologyNeed">
             <property name="text">
              <string>Radiology Need</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxPharmecyNeed">
             <property name="text">
              <string>Pharmecy Need</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxRoomNeed">
             <property name="text">
              <string>Need Room</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="40,50">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_9">
           <property name="title">
            <string>Find Personal</string>
           </property>
           <widget class="QTextEdit" name="textEditPersonalData">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>261</width>
              <height>121</height>
             </rect>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>14</x>
              <y>30</y>
              <width>251</width>
              <height>24</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Pesrsona ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPersonalID"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonFindPersonal">
               <property name="text">
                <string>Find</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Registration Personal</string>
           </property>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>80</y>
              <width>271</width>
              <height>121</height>
             </rect>
            </property>
            <property name="title">
             <string>Time Shift</string>
            </property>
            <widget class="QCheckBox" name="checkBoxSat">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>72</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>Sat</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBoxSun">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>72</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>Sun</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBoxMon">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>60</y>
               <width>72</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>Mon</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBoxTue">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>80</y>
               <width>72</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>Tue</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBoxWed">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>20</y>
               <width>72</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>Wed</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBoxThu">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>40</y>
               <width>72</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>Thu</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBoxFri">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>60</y>
               <width>72</width>
               <height>19</height>
              </rect>
             </property>
             <property name="text">
              <string>Fri</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBoxNight">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>20</y>
               <width>101</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Night Shift</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBoxAfternoon">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>40</y>
               <width>101</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Afternoon Shift</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBoxMorning">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>60</y>
               <width>101</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Morning Shift</string>
             </property>
            </widget>
           </widget>
           <widget class="QComboBox" name="comboBoxPersonalSpecialist">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>55</y>
              <width>201</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>58</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Specialist</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>26</y>
              <width>31</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditRegPersonalName">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>26</y>
              <width>91</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>148</x>
              <y>26</y>
              <width>21</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditRegPersonalID">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>26</y>
              <width>101</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonCreatPersonal">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>210</y>
              <width>80</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Creat</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout" columnstretch="20,90">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Room Status</string>
         </property>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Rooms:</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelAllRooms">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>48</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Fiiled Rooms:</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelUsedRooms">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>50</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>67</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Free Room:</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelFreeRooms">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>70</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonUpdata">
          <property name="geometry">
           <rect>
            <x>81</x>
            <y>100</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Recived Survices</string>
         </property>
         <widget class="QCheckBox" name="checkBoxSurgeryUsed">
          <property name="geometry">
           <rect>
            <x>169</x>
            <y>22</y>
            <width>72</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Surgery</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBoxDantisUsed">
          <property name="geometry">
           <rect>
            <x>169</x>
            <y>42</y>
            <width>72</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Dentis</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBoxRadiologyUsed">
          <property name="geometry">
           <rect>
            <x>169</x>
            <y>62</y>
            <width>72</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Radiology</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBoxParmecyUsed">
          <property name="geometry">
           <rect>
            <x>169</x>
            <y>82</y>
            <width>72</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Pharmecy</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBoxRoomUsed">
          <property name="geometry">
           <rect>
            <x>169</x>
            <y>102</y>
            <width>72</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Room Assgind</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Patient ID</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonRecivedSurvices">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>80</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Recived</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="textEditRecived">
          <property name="geometry">
           <rect>
            <x>255</x>
            <y>30</y>
            <width>371</width>
            <height>91</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="labelPatientID_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>30</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonClear">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>80</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Clear All</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHospital_Automation">
    <property name="title">
     <string>Hospital Automation</string>
    </property>
   </widget>
   <addaction name="menuHospital_Automation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
